docker-compose.yaml

```
@@ -0,0 +1,94 @@
+version: "3.6"
+services:
+  # CB-Spider
+  cb-spider:
+    image: cloudbaristaorg/cb-spider:0.9.0
+    container_name: cb-spider
+    platform: linux/amd64
+    ports:
+      - "1024:1024"
+      - "2048:2048"
+    volumes:
+      # - ./conf/log_conf.yaml:/root/go/src/github.com/cloud-barista/cb-spider/conf/log_conf.yaml
+      - ./conf/store_conf.yaml:/root/go/src/github.com/cloud-barista/cb-spider/conf/store_conf.yaml
+      - ./container-volume/cb-spider-container/meta_db/:/root/go/src/github.com/cloud-barista/cb-spider/meta_db/
+      - ./container-volume/cb-spider-container/log/:/root/go/src/github.com/cloud-barista/cb-spider/log/
+    environment:
+      - PLUGIN_SW=OFF
+      # - SERVER_ADDRESS=localhost
+      # if you leave these values empty, REST Auth will be disabled.
+      - API_USERNAME=
+      - API_PASSWORD=
+      - SPIDER_LOG_LEVEL=error
+      - SPIDER_HISCALL_LOG_LEVEL=error
+      - ID_TRANSFORM_MODE=ON
+    healthcheck: # for CB-Spider
+      test: [ "CMD", "curl", "-f", "http://localhost:1323/tumblebug/readyz" ]
+      interval: 5s
+      timeout: 1s
+      retries: 3
+      start_period: 5s
+
+  # CB-Tumblebug
+  cb-tumblebug:
+    image: cloudbaristaorg/cb-tumblebug:0.9.0
+    # build:
+    #   context: .
```

```suggestion
    build:
      context: .
```
docker-compose up --build 했을때, 공용 컨테이너 이미지가 아니라, 현재 소스기준으로 컨테이너 구동하게 하려고 하셨던거죠?
활성화하면 매우 유용할 것 같습니다.